sygnal/webpushpushkin.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, name, sygnal, config):
             raise PushkinSetupException("path in 'vapid_private_key' does not exist")
         try:
             self.vapid_private_key = Vapid.from_file(private_key_file=privkey_filename)
-        except BaseException as e:
```

Comment:
I had this first fwiw, but switched to the more general one because I assumed [open and read here](https://github.com/web-push-libs/vapid/blob/main/python/py_vapid/__init__.py#L118) could throw some kind of I/O exception?

Reply:
Ah, it's merged already :tada:  probably not too important in any case. Thanks for seeing this through!

Reply:
> I had this first fwiw, but switched to the more general one because I assumed [open and read here](https://github.com/web-push-libs/vapid/blob/main/python/py_vapid/__init__.py#L118-L119) could throw some kind of I/O exception?

They could, but catching `BaseExeption` is usually not great. We check that the file exists above so I suspect that wouldn't happen...although I guess there could be permission issues.